packages/salesforcedx-vscode-apex-replay-debugger/src/activation/getDialogStartingPath.ts

```diff
@@ -10,7 +10,7 @@ import {
   projectPaths,
   workspaceUtils
 } from '@salesforce/salesforcedx-utils-vscode';
-import * as pathExists from 'path-exists';
+import * as fs from 'fs';
```

Comment:
nit: if you just need pathExists it's typically better to limit the import to just that function
`import { existsSync } from 'fs'`